string roi_id
int16 start
int16 end
---
bool success
